AirCasting/CoreData/Storages/UIStorage.swift

```diff
@@ -66,6 +66,49 @@ final class HiddenCoreDataUIStorage: UIStorageContextUpdate {
         sessionEntity.userInterface?.sensorName = stream
     }
     
+    // MARK: Session ordering
+    
+    func updateSessionOrder(_ order: Int, for sessionUUID: SessionUUID) {
+        do {
+            guard let session = try context.existingSessionable(uuid: sessionUUID) else {
+                Log.error("Trying to update order for nonexistent session [\(sessionUUID)]")
+                return
+            }
+            createUIStateIfNeededForSessionable(session)
+            session.userInterface?.rowOrder = Int64(order)
+            try context.save()
+        } catch {
+            Log.error("Error when saving changes in session: \(error.localizedDescription)")
+        }
+    }
+
+    func giveHighestOrder(to sessionUUID: SessionUUID) {
+        do {
+            guard let session = try context.existingSessionable(uuid: sessionUUID) else {
+                Log.error("Trying to update order for nonexistent session [\(sessionUUID)]")
+                return
+            }
+            let highestOrder = try context.getHighestRowOrder()
+            session.userInterface?.rowOrder = (highestOrder ?? 0) + 1
```

Comment:
We need to add `createUIStateIfNeededForSessionable(session)` above, because userInterface gets deleted when a session is unfollowed (probably something we want to change) and so adding this highest order doesn't work for sessions that had previously been followed